infrastructure/README.md

```diff
@@ -0,0 +1,56 @@
+# AIDA Interchange Format Infrastructure
+
+## CloudFormation
+
+The AIDA Interchange Format project supports creating the AIDA validation environment on AWS via CloudFormation scripts. Read below on how to validate, create, and delete the various stacks necessary to execute AIDA validations. 
+
+### Validation
```

Comment:
How about including something about installing the AWS CLI (with link to [its instructions](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html))?  It could be a new section called `Prerequisites`, or could just be included in this section.